chamge/amqp-edge-backend.c

```diff
@@ -11,9 +11,20 @@
 #include <gio/gio.h>
 #include <amqp.h>
 #include <amqp_tcp_socket.h>
+#include <json-glib/json-glib.h>
 
 #define AMQP_EDGE_BACKEND_SCHEMA_ID "org.hwangsaeul.Chamge1.Edge.AMQP"
 #define DEFAULT_CONTENT_TYPE "application/json"
+#define DEFAULT_CONTENT_TYPE_LEN 16
+
+typedef enum
+{
+  CHAMGE_AMQP_EDGE_BACKEND_NULL = 0,
+  CHAMGE_AMQP_EDGE_BACKEND_ENROLLED,
+  CHAMGE_AMQP_EDGE_BACKEND_ACTIVATED,
+  CHAMGE_AMQP_EDGE_BACKEND_PLAYING,
+  CHMAGE_AMQP_EDGE_BACKEND_ERROR
+} ChamgeEdgeStatus;
```

Comment:
I don't think we need to define backend status. Instead please consider using `ChamgeNodeState` only.